sysdeps/managarm/generic/drm.cpp

```diff
@@ -1209,6 +1209,12 @@ int ioctl_drm(int fd, unsigned long request, void *arg, int *result, HelHandle h
 			                    << frg::endlog;
 			return 0;
 		}
+		case DRM_IOCTL_WAIT_VBLANK: {
```

Comment:
How spammy is this? Also, implementing this probably isn't hard as we can already await them on file read.

Reply:
Not that spammy at all if I recall correctly.